common-java-libs/org.wso2.apk.apimgt.rest.api.admin.v1.common/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id "org.openapi.generator" version "6.2.1"
```

Comment:
Shall we use single quotes for those don't have variable names?

Reply:
Fixed with [7804ebd](https://github.com/wso2/apk/pull/110/commits/7804ebd19eea6fb6da6fb2d3cfdb7394a003f985)